### PR TITLE
increment version number

### PR DESCRIPTION
**Pull request by animanathome**
1.0.7 already exists ...

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@lumen5/framefusion",
-  "version": "1.0.7",
+  "version": "1.0.8",
```
Comment (michaeljherrmann) [inline package.json:3]: I don't see a tag for 1.0.8 in framefusion?

Comment (michaeljherrmann) [reply package.json:3]: oh wait this is the framefusion repo, I get it... 😅

Comment (animanathome) [reply package.json:3]: it's here: https://www.npmjs.com/package/@lumen5/framefusion

Comment (animanathome) [reply package.json:3]: we need to run `npm publish`

Comment (michaeljherrmann) [reply package.json:3]: Did the ci job fail last time then?

**Pull request merged**